src/applications/ezr/tests/unit/utils/helpers/form-config.unit.spec.js

```diff
@@ -279,4 +286,220 @@ describe('ezr form config helpers', () => {
       });
     });
   });
+
+  context('when `includeGulfWarServiceDates` executes', () => {
+    context(
+      'when the `gulfWarService` value is true, `includeTeraInformation` evaluates to true, ' +
```

Comment:
Would it make sense to add a spec that has a dynamic date that is right at today - 15 years? Maybe that's a little testing the logic with the same log so maybe it's fine?

Reply:
I think that makes sense, sure! I went ahead and added a test for each function that uses the "present day - 15 years" logic!